src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
-}
+  const robot = {
+    _name: name,
+    _wheels: wheels,
+    _version: version,
```

Comment:
Мені підказали що можна укоротити запис ось так, тобто воно буде одразу і створювати ключ і приймати значеченняю
```suggestion
    name,
    wheels,
    version,
```

Reply:
Дякую! Класна опція